src/azure-cli/azure/cli/command_modules/servicefabric/custom.py

```diff
@@ -1472,7 +1471,6 @@ def _download_secret(cli_ctx, vault_base_url, secret_name, pem_path, pfx_path, s
         try:
             import base64
             decoded = base64.b64decode(secret_value)
-            p12 = pkcs12.load_pkcs12(decoded, None)
```

Comment:
Will this deletion cause problem?

Reply:
It's marked as unused in PyCharm. I can add it back in.